test/Sentry.Tests/SentryMonitorOptionsTests.cs

```diff
@@ -0,0 +1,112 @@
+
+        Special characters are:
+            *	any value
+            ,	value list separator
+            -	range of values
+            /	step values
+    */
+    [Theory]
+    [InlineData("* * * * *")]
+    [InlineData("0 0 1 1 *")]
+    [InlineData("0 0 1 * 0")]
+    [InlineData("59 23 31 12 7")]
+    [InlineData("0 */2 * * *")]
+    [InlineData("0 8-10 * * *")]
+    [InlineData("0 6,8,9 * * *")]
+    // Step values (*/n)
+    [InlineData("*/15 * * * *")]        // Every 15 minutes
+    [InlineData("0 */6 * * *")]         // Every 6 hours
+    [InlineData("0 0 */2 * *")]         // Every 2 days
+    [InlineData("0 0 1 */3 *")]         // Every 3 months
+    // Complex ranges
+    [InlineData("1-15 * * * *")]        // Minutes 1 through 15
+    [InlineData("* 9-17 * * *")]        // Business hours
+    [InlineData("* * 1-15,16-31 * *")]  // Split day ranges
+    // Multiple comma-separated values
+    [InlineData("1,15,30,45 * * * *")]  // Specific minutes
+    [InlineData("* 9,12,15,17 * * *")]  // Specific hours
+    [InlineData("0 0 * * 1,3,5")]       // Monday, Wednesday, Friday
+    // Combinations of special characters
+    [InlineData("*/15 9-17 * * 1-5")]   // Every 15 min during business hours on weekdays
+    [InlineData("0 8-10,13-15 * * *")]  // Morning and afternoon ranges
+    // Edge cases
+    [InlineData("0 0 1 1 0")]           // Minimum values
+    [InlineData("*/1 * * * *")]         // Step of 1
+    [InlineData("* * 31 */2 *")]        // 31st of every other month
+    public void Interval_ValidCrontab_DoesNotThrow(string crontab)
+    {
+        // Arrange
+        var options = new SentryMonitorOptions();
+
+        // Act
+        options.Interval(crontab);
+    }
+
+    [Fact]
+    public void Interval_SetMoreThanOnce_Throws()
+    {
+        // Arrange
+        var options = new SentryMonitorOptions();
+
+        // Act
+        options.Interval(1, SentryMonitorInterval.Month);
+        Assert.Throws<ArgumentException>(() => options.Interval(2, SentryMonitorInterval.Day));
+    }
+
+    [Theory]
+    [InlineData("")]
+    [InlineData("not a crontab")]
+    [InlineData("* * a * *")]
+    [InlineData("60 * * * *")]
+    [InlineData("* 24 * * *")]
+    [InlineData("* * 32 * *")]
+    [InlineData("* * * 13 *")]
+    [InlineData("* * * * 8")]
+    // Invalid step values
+    [InlineData("*/0 * * * *")]         // Step value cannot be 0
+    [InlineData("*/100 * * * *")]       // Step value too large for minutes
+    [InlineData("* */25 * * *")]        // Step value too large for hours
+    [InlineData("* * */32 * *")]        // Step value too large for days
+    [InlineData("* * * */13 *")]        // Step value too large for months
+    [InlineData("* * * * */8")]         // Step value too large for weekdays
+    [InlineData("*/60 * * * *")]        // Step value equals max value for minutes
+    [InlineData("* */24 * * *")]        // Step value equals max value for hours
```

Comment:
question: compatibility / format

Are these values rejected by Sentry?
https://crontab.guru/ says that the strings of `// Invalid step values` (except the first one) are actually valid.
I am using cron tables super rarely ... but are there different subsets of valid cron table formats?

Reply:
Yeah you're right... Sentry seems quite flexible regarding step values:
![image](https://github.com/user-attachments/assets/59d803dc-e4df-4d55-8fe2-b4c75de458db)

So it seems the step values can be anything (as long as they're greater than 0).

The ranges and list values have to be valid for the field type though.

I'll update the tests and the regex accordingly.